test/liquid_handle_unit_test.py

```diff
@@ -420,3 +421,280 @@ def test_has_calibration_with_calibration(self):
         )
         self.mix._liquid = calibrated_lc
         assert self.mix._has_calibration() is True
+
+
+class ProteinBuffer(LiquidClass):
+    def __init__(self):
+        super(ProteinBuffer, self).__init__()
+        self.name = "protein_buffer"
+
+
+class LiquidHandleTester(object):
+    @pytest.fixture(autouse=True)
+    def setup(self):
+        self.protocol = Protocol()
+        self.tube = self.protocol.ref("tube", cont_type="micro-2.0", discard=True)
+        self.tube.well(0).set_volume("50:microliter")
+        self.flat = self.protocol.ref("flat", cont_type="96-flat", discard=True)
+
+
+class TestLiquidHandleDispenseMode(LiquidHandleTester):
+    def test_location_count(self):
+        volume = "5:uL"
+        instruction = self.protocol.liquid_handle_dispense(
+            source=self.tube.well(0),
+            destination=self.flat.wells_from(0, 12),
+            volume=volume,
+        )
+
+        assert len(instruction.data["locations"]) == 15
```

Comment:
I would argue that we should have a test similar to this in `test_protocol.py`, where we simply test the `liquid_handle_dispense` method. Then we have these other LiHa-specific tests here in lihaunit test

Reply:
🤔 I see other tests for this method below that i would also want in `test_protocol`, so I'm thinking whether or not we should move all of them, especially that only test this one method, into that other test file.

Thoughts?

Reply:
@joshhacksthings (discussion)
Options:
1. Move liquid_handle_dispense() tests to protocol_test.py
2. keep them in liquid_handle_unit_test.py and put 1-2 in protocol_test.py
3. move them to a new file called: liquid_handle_dispense_test.py

Downsides:
1. This test file is already very long and it will just become more difficult to pinpoint dependencies for different test classes in same file
2. Testing one method in two different locations.
3. Extra test file.

Upsides:
1. Not really sure the advantage of option 1
2. perhaps different environments provides more robust testing
3. Isolated tests. Easy to modify without breaking other tests accidentally.

Reply:
@joshhacksthings sidenote: 
ran athae_workflow protocol using old txappy (version 0.11.1) and old appy(7.7.0)
ran athae_workflow protocol using new txappy (version 2.0.0) and new appy(7.8.0)

**There was no difference in json output between the two.** Pretty confident there are no naming issues.
I did this using the local install of the master branches of these repos vs the feature branches for comparison.
This is because the versions released are not exactly the same as the latest version on github.

Also checked gappy. Clear for launch! Just need you to respond to above comment

Reply:
@joshhacksthings if using latest versions of txappy and appy, make sure to update the first line of: internal_protocols/internal_protocols/promodules/dispense/dispense_reagents.py

from
from txappy.liquid_handle.dispense import Dispense
to
from autoprotocol.liquid_handle.dispense import Dispense